linkis-computation-governance/linkis-entrance/src/main/scala/org/apache/linkis/entrance/conf/EntranceConfiguration.scala

```diff
@@ -225,4 +225,7 @@ object EntranceConfiguration {
   val CREATOR_IP_SWITCH =
     CommonVars("wds.linkis.entrance.user.creator.ip.interceptor.switch", false)
 
+  val SEND_INSTANCE =
+    CommonVars("wds.linkis.entrance.whether.open.send.instance", true)
```

Comment:
new configuration need to record at docs/configuration and  docs/info-1.3.2.md

Reply:
done